lib/egon/undercloud/commands.rb

```diff
@@ -329,6 +328,9 @@ def self.OSP8_no_registration_no_packages
         echo '  RedisPassword: Rhq8Fd7eEIoPP821Ui' >> openstack-tripleo-heat-templates/overcloud-resource-registry-puppet.yaml
         echo '  HAProxyStatsPassword: veLYjyhxgs7GtQuKXF' >> openstack-tripleo-heat-templates/overcloud-resource-registry-puppet.yaml
 
+        echo 'parameter_defaults:' >> openstack-tripleo-heat-templates/environments/deployment_parameters.yaml
```

Comment:
It's correct, the output should be something like:

parameter_defaults:
  ComputeCount: '2'